Regression/WarpX-tests.ini

```
@@ -2308,7 +2308,23 @@ useOMP = 1
 numthreads = 2
 compileTest = 0
 doVis = 0
-analysisRoutine = Examples/Modules/nuclear_fusion/analysis_deuterium_tritium_fusion.py
+analysisRoutine = Examples/Modules/nuclear_fusion/analysis_two_product_fusion.py
+
+[Deuterium_Deuterium_Fusion_3D]
+buildDir = .
+inputFile = Examples/Modules/nuclear_fusion/inputs_deuterium_deuterium_3d
+runtime_params = warpx.do_dynamic_scheduling=0 warpx.serialize_initial_conditions=1
+dim = 3
+addToCompileString =
+cmakeSetupOpts = -DWarpX_DIMS=3
+restartTest = 0
+useMPI = 1
+numprocs = 2
+useOMP = 1
+numthreads = 2
```

I think we need to set `numthreads = 1` if `numprocs = 2`, because of the limited CI resources:
```suggestion
numthreads = 1
```